sdks/java/core/pom.xml

```diff
@@ -372,6 +372,12 @@
     </dependency>
 
     <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-lang3</artifactId>
+      <version>3.5</version>
```

Comment:
to the extent possible, no version numbers here. Move to `dependencyManagement` in root pom.